test/AbstractMemoryTest.cpp

```diff
@@ -0,0 +1,209 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (July of 2023,
+// schlegea@informatik.uni-freiburg.de)
+
+#include <absl/strings/str_cat.h>
+#include <gtest/gtest.h>
+
+#include <algorithm>
+#include <cstdint>
+#include <ranges>
+#include <vector>
+
+#include "util/AbstractMemory/Memory.h"
+
+// Importing all the literals.
+using ad_utility::operator""_Byte;
+using ad_utility::operator""_KB;
+using ad_utility::operator""_MB;
+using ad_utility::operator""_GB;
+using ad_utility::operator""_TB;
+using ad_utility::operator""_PB;
```

Comment:
The typical pattern here is to put all of those in a dedicated namespace (e.g. `ad_utility::memory_literals`) and the do a `using namespace`.

Reply:
Done, see [b548d0a](https://github.com/ad-freiburg/qlever/pull/1032/commits/b548d0a6b1c8f517b8cf776500015a79e78e2a75).